### PR TITLE
docs(TextArea): move to storybook

### PR DESCRIPTION
**Pull request by rodleyorosa**
Refs: CDS-284

### CODE REVIEW

File: src/components/inputs/textArea/TextArea.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+type Story = StoryObj<typeof TextArea>;
+
+export const Default = {} satisfies Story;
+
+export const Label = {
+	args: {
+		label: 'This is the Input label'
+	}
+};
```
Comment (beawar) [inline src/components/inputs/textArea/TextArea.stories.tsx]: `satisfies Story` is missing (also in most of next ones)

File: src/components/inputs/textArea/TextArea.mdx
Comment (beawar) [inline src/components/inputs/textArea/TextArea.mdx:1]: We should add the Figma component as done in the Snackbar. It is useful to keep track of both the component and the ui kit

File: src/components/inputs/textArea/TextArea.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+export const BorderColor = {
+	args: {
+		borderColor: 'warning'
+	}
+};
+
+export const UncontrolledMode = {
+	render: UncontrolledTextArea
```
Comment (beawar) [inline src/components/inputs/textArea/TextArea.stories.tsx:65]: I think we should render the source for the rendered component, as done in the ChipInput.
For these kind of stories, where we are showing how the component should be used, I think it is better to show the implemented code.
Valid also for the controlled mode story.

File: src/components/inputs/textArea/TextArea.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+<Canvas of={TextAreaStories.TextColor} />
+
+### BorderColor
+
+<Canvas of={TextAreaStories.BorderColor} />
+
+### UncontrolledMode
+
+<Canvas of={TextAreaStories.UncontrolledMode} source={{ code: UncontrolledModeCode }} />
```
Comment (beawar) [inline src/components/inputs/textArea/TextArea.mdx:54]:
```suggestion
<Canvas of={TextAreaStories.UncontrolledMode} source={{ code: UncontrolledModeCode, language: 'tsx' }} />
```

File: src/components/inputs/textArea/TextArea.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+### ControlledMode
+
+<Canvas of={TextAreaStories.ControlledMode} source={{ code: ControlledModeCode }} />
```
Comment (beawar) [inline src/components/inputs/textArea/TextArea.mdx:58]:
```suggestion
<Canvas of={TextAreaStories.ControlledMode} source={{ code: ControlledModeCode, language: 'tsx' }} />
```

**Review by beawar — COMMENTED** [review]
LGTM, ~~just add the language to the source components (default is jsx, with tsx types are better colored in the final source snippet)~~

EDIT: We can set the language globally at preview level. Will do in a next pr

**Review by beawar — DISMISSED** [review]

**Review by beawar — APPROVED** [review]

**Review by CataldoMazzilli — APPROVED** [review]

**Pull request merged**